jbpm-wb-integration/jbpm-wb-integration-client/src/main/java/org/jbpm/workbench/wi/client/editors/deployment/descriptor/sections/requiredroles/RequiredRolesListItemView.html

```diff
@@ -17,11 +17,11 @@
 <table>
     <tbody>
     <tr data-field="root">
-        <td class=kie-tablelist__cell">
-            <span data-field="role"></span>
+        <td class="kie-tablelist__cell col-md-10">
```

Comment:
and reviewing these, better to use _100%_ width directly in here instead of _col-md-10_

Reply:
directly add style="width"?

Reply:
but the row also have delete button, also use 100%?